solutions/fully-configurable/variables.tf

```diff
@@ -512,3 +512,121 @@ variable "ssh_key_secret_name" {
   nullable    = false
   description = "The name for the secret created for the auto generated ssh private key."
 }
+
+##############################################################################
+# Logging Agent Variables
+########################################################################################################################
+
+variable "install_logging_agent" {
+  type        = bool
+  default     = false
```

Comment:
Due to this being opt-in for both module and DA, we should have at least one test case that deploys the agents?